.github/workflows/e2e-browser.yml

```diff
@@ -1,72 +1,68 @@
-name: End-to-end tests (browser)
+name: End-to-end browser tests
 
-on: 
-  deployment_status:
-    branches-ignore:
-      - 'dependabot/npm_and_yarn/**'
-  workflow_dispatch:
+on: [push, workflow_dispatch]
 
 env:
   CI: true
 jobs:
-  test:
-    runs-on: ubuntu-20.04
-    if: ${{github.event.deployment_status.state == 'success'}}
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # Available OS's: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+        os: [ubuntu-20.04]
+        node-version: [14.x]
     steps:
       - uses: actions/checkout@v2.3.4
-      - name: Use Node.js 12.x
+      - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v2.1.5
+        with:
+          node-version: ${{ matrix.node-version }}
       - name: Cache node modules
         uses: actions/cache@v2.1.5
         env:
           cache-name: cache-node-modules
         with:
-          path: e2e/browser/node_modules
+          path: node_modules
           key: ${{ runner.os }}-node${{ runner.node-version }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-      - name: Run tests
-        # Dependabot cannot access secrets, so it doesn't have a token to authenticate to ESS.
-        # Since all the other jobs of this workflow depend on this one, skipping it should
-        # skip the entire workflow.
-        if: ${{ github.actor != 'dependabot[bot]' }}
+      - run: npm ci
+      - run: npm run bootstrap -- --ci
+      - run: npm run build
+      - name: Prepare browser-based end-to-end tests
+        run: |
+          cd packages/browser/examples/demoClientApp
+          npm install
+          cd ../../../../
+      - name: Run browser-based end-to-end tests (Linux)
+        # TODO: Add Edge/merge with Windows setup once Edge is available on Linux:
```

Comment:
Looks like you're just running on Ubuntu, so no Windows setup to merge it with :)

```suggestion
```